### DISCUSSION TITLE
Define SQL table name

### DISCUSSION THREAD

**Discussion by mdmoura** [quartznet/quartznet#961] [General]
Could an option be added to change the SQL table name use by Quartz and not only the prefix?

**Comment by lahma** [comment]
What is the use case? The prefix should be quite enough for the need to separate from other tables, if there are any. Usually dedicating database for Quartz use is a good idea if there's a lot triggers firing. Also mixing with "business database" can make backups larger and thus cause problems when database is unavailable.

**Reply by mdmoura** [reply]
I asked out of curiosity. Sooner or later might be the need to have custom tables' names for one reason or another ... 

At least is what I've been experience in a few projects with other libraries ...

**Reply by lahma** [reply]
Well let's hope that there's no such need in the future. We've been lucky enough to live without such configurability for over 10 years 🙂
